src/actions/friendsServices.ts

```diff
@@ -0,0 +1,30 @@
+import { UserProfileData } from 'types/UserProfileData'
+import axiosInstance from './axiosInstance'
+
+export const getFriends = async (url: string) => {
+  try {
+    const response = await axiosInstance.get(url)
+    return response.data as Array<UserProfileData>
```

Comment:
better way to type response in further way:
type GetFriendsResponse = {...}
`export const getFriends = async (url: string):Promise<GetFriendsResponse>`

Reply:
Added